scripts/setup/install-python

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Use this script to install python using conda.
+set -e
+set -x
+
+# Defaults
+PYTHON_VERSION=3.7
+CONDA_VENV_PATH=~/.conda/envs/zulip-conda-venv
```

Comment:
This path isn't going to be viable in production; I think?  We'll want a path under e.g. `/srv` that can be used by multiple user accounts, I think.

Reply:
I wanted to use `/srv` but ran into permission issues. I will retry.